sunpy/map/sources/tests/test_hmi_synoptic_source.py

```diff
@@ -0,0 +1,76 @@
+"""Test cases for SDO Map subclasses.
+This particular test file pertains to HMIMap.
+@Author: Pritish C. (VaticanCameos)
+"""
+
+import os
+import glob
+import numpy as np
+import pytest
+
+from sunpy.map.sources.sdo import HMISynopticMap, HMIMap
+from sunpy.map import Map
+from sunpy.util import MetaDict
+
+
+@pytest.fixture
+def hmi_synoptic():
+    # sample HMI synoptic map
+    # Please do not edit this header, it is a 1:1 copy of a HMI map header
+    header = MetaDict([
```

Comment:
It would be better to extract the full fits header as ascii I think, you can do that with the `fitsheader` command provided by astropy. See https://github.com/sunpy/sunpy/blob/master/sunpy/visualization/animator/tests/test_wcs.py#L15 for an example